solution.txt

```diff
@@ -142,23 +141,23 @@ Exercise 12:
 */
 
 // Complete Exercise 12 below...
-
-let fizz = [];
-let buzz = [];
-let fizzbuzz = [];
-
-nums.forEach(fuzzbizz);
-
-function fuzzbizz(num){
-  if (num % 3 === 0 && num % 5 === 0) {
-    fizzbuzz.push(num);
-  } else if (num % 3 === 0) {
-    fizz.push(num);
-  } else if (num % 5 === 0) {
-    buzz.push(num);
+const fizz = [];
+const buzz = [];
+const fizzbuzz = [];
+const addNumberToFizzBuzzArrays = (number) => {
+  if (number % 3 === 0 && number % 5 === 0){
+    fizzbuzz.push(number);
+  }
+  if (number % 3 === 0) {
+    fizz.push(number);
+  }
+  if (number % 5 === 0) {
+    buzz.push(number);
   }
 }
 
+nums.forEach(addNumberToFizzBuzzArrays);
```

Comment:
💯 yes! nice work figuring out that the conditionals need to check against 3 AND 5 together *first*! If you were to start with checking against 3 only, all the numbers divisible by 3 AND 5 would only get caught as being divisible by 3.

Also, you used only `if` statements, and no `else if` statements. Because of this, each number gets checked against each `if`. If you had done `if() {} else if() {}` etc., each number would only have been put into one array instead of multiple arrays. 

Let me know if this all makes sense!